### PR TITLE
Allow forking AtomOne

### PR DESCRIPTION
**Pull request by jaekwon**
This makes clear that anyone can launch a chain from the ideas collected here.

### CODE REVIEW

**Comment by moul** [comment]
I recommend adding a LICENSE.md file before merging this one.

In your new paragraph, consider explicitly requesting the license choice, such as Apache2.0 or more liberal licenses, or *GPL* and GnoPL licenses.

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@ _This document is a work in progress, please make PRs_ _What matters at this
 very moment is that we open ISSUEs and begin discussions about every facet that
 should go in the founding documents_
 
+_ALL CONTRIBUTIONS TO THIS REPO, ITS ISSUES, PROJECTS, AND DISCUSSIONS MAY BE USED
+IN ANY EXPLICIT GITHUB FORK WITH A NEW AND DISTINCT NAME TO LAUNCH ANY FORK OR SPLIT
+THAT MODIFIES ANY OF THE IDEAS MENTIONED HERE WITH FULL ATTRIBUTION._
```
Comment (moul) [inline README.md:7]:
```suggestion
_ALL CONTRIBUTIONS TO THIS REPO, ITS ISSUES, PROJECTS, AND DISCUSSIONS MAY BE USED
IN ANY EXPLICIT GITHUB FORK WITH A NEW AND DISTINCT NAME TO LAUNCH ANY FORK OR SPLIT
THAT MODIFIES ANY OF THE IDEAS MENTIONED HERE. THIS IS ALLOWED WITH FULL ATTRIBUTION,
PROVIDED THAT THE FORK ALSO MAINTAINS A LICENSE THAT GRANTS SIMILAR OR GREATER
FREEDOMS, SUCH AS APACHE 2.0, OTHER MORE LIBERAL LICENSES, OR *GPL* AND GnoPL LICENSES._
```

**Comment by moul** [comment]
The aim is to encourage positive competition. Forking is not just about competing hard. It's about trying new things in different ways. Forks can also work together and help each other.

We aim to promote a culture where forking, including forking from forks and subsequent iterations, is a viable option for anyone seeking to fix or just improve a current version. 

It's ideal to have a core set of rules or a simple constitution that stays relevant for a long time, like 100 years or through 20 rounds of forking. However, we should also be ready for the idea that these rules might one day be outdated. This mindset prepares us for ongoing changes and growth. For the next decade(?), let's apply this constitution across our network of related forks, among a community of cooperative builders and members.

**Comment by jaekwon** [comment]
Specified the GnoNGPL license.

**Pull request merged**